.circleci/config.yml

```diff
@@ -94,6 +94,21 @@ jobs:
               done
               exit $ret
             fi
+  build-push-community-platform:
+    executor: ubuntu-medium
+    steps:
+      - checkout
+      - aws-setup
+      - aws-ecr-setup
+      - run: echo $GITHUB_CONTAINER_REGISTRY_TOKEN | docker login ghcr.io -u USERNAME --password-stdin
+      - run:
+          name: Build or skip
+          shell: /bin/bash
+          command: |
+            set -e
+            cd ecosystem/platform/server
+            docker buildx create --use
+            GIT_SHA1=${CIRCLE_SHA1} docker buildx bake --progress=plain --push -f ./docker-bake.hcl
```

Comment:
nit: could you name the image `dev_$(git rev-parse --short=8 HEAD)` for consistency?

Reply:
hmm, I'm actually using the full sha1 intentionally here. From my experience it's easier to just use the full sha1 everywhere as identifier and in links since in some contexts / tools only that one is permitted and github links and it's "copy" menu use the full sha1 by default as well.
I think we should actually move to using the full git sha1 for all images in future too.
I'm gonna keep the full sha1 here for now but we can have a discussion about this again tomorrow if you think otherwise.